README.md

```diff
@@ -24,11 +24,11 @@ repository, you must select the appropriate one:
 | Open edX release | Tutor version | XBlock version | XBlock branch |
 |------------------|---------------|----------------|---------------|
 | Maple            | `>=13.2, <14` | `>=6.0, <7.0`  | `stable-6.0`  |
-| Nutmeg           | `>=14.0, <15` | `>=7.0`        | `master`      |
-| Olive            | `>=15.0, <16` | `>=7.5`        | `master`      |
-| Palm             | `>=16.0, <17` | `>=7.5`        | `master`      |
-| Quince           | `>=17.0, <18` | `>=7.9`        | `master`      |
-| Redwood          | `>=18.0, <19` | `>=7.12`       | `master`      |
+| Nutmeg           | `>=14.0, <15` | `>=7.0, <8.0`  | `stable-7.0`  |
+| Olive            | `>=15.0, <16` | `>=7.5, <8.0`  | `stable-7.0`  |
+| Palm             | `>=16.0, <17` | `>=7.5, <8.0`  | `stable-7.0`  |
+| Quince           | `>=17.0, <18` | `>=7.9, <8.0`  | `stable-7.0`  |
```

Comment:
`stable-7.0` doesn't work for a branch where the latest minor release is 7.13. :) You probably want to call the branch `stable-7` instead.